src/main/java/org/wso2/carbon/inbound/amazonsqs/AmazonSQSPollingConsumer.java

```diff
@@ -305,6 +311,32 @@ private boolean isRollback(org.apache.synapse.MessageContext msgCtx) {
         return false;
     }
 
+    /**
+     * Validates the credential inputs. If both the secret key and the access key are provided or
+     * if none of them are provided, it will be considered as a valid combination. Providing only one of
+     * them will be an invalid combination.
+     *
+     * @param secretKey the key that is used to sign requests
+     * @param accessKey the key that corresponds to the secret key that is used to sign the request
+     * @return true if both the secret key and the access key are provided or none of them are provided. Throws an
+     * runtime exception if only one is provided.
+     */
+    private boolean validateCredentialsInput(String secretKey, String accessKey) {
```

Comment:
Since the return value of the method is never used, Shall we have this as void?

Reply:
Done